src/main/java/hudson/plugins/accurev/CheckForChanges.java

```diff
@@ -295,4 +192,42 @@ public static boolean checkStreamForChanges(AccurevServer server,
         }
         return isTransLatestThanBuild;
     }
+
+    private static boolean changesMatchFilter(TaskListener listener, Collection<String> serverPaths, List<String> filters) {
+        if (filters == null || filters.isEmpty()) {
+            // No filters, so always a match.
+            return true;
+        }
+
+        for (String filterPath : filters) {
+            // Paths are sanitized using java.nio.file.Paths (makes sure that the same path separators are used)
+            final Path fp = Paths.get(filterPath);
+            for (String serverPath : serverPaths) {
+                final Path sp = Paths.get(serverPath);
```

Comment:
Consider using `final Path sp = Paths.get(serverPath.replace("\\", "/"));`
While getListOfPollingFilters returns a list where it has replaces backslash with forward slash
To later those with those from the serverPath which also gets replaced with forward slash.
So in the end it uses the same file separator.